protocols/upnp/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.2.1
+- Ignore oneshot sender error when behaviour has dropped.
+  See [PR 5096](https://github.com/libp2p/rust-libp2p/pull/5096).
```

Comment:
Please reword this such that it tells a user what was actually changed. The bug we are fixing here is a panic, right? That is what is important. How that is fixed is an implementation detail :)